editor/plugins/script_editor_plugin.cpp

```diff
@@ -486,6 +487,31 @@ void ScriptEditor::_clear_execution(REF p_script) {
 	}
 }
 
+void ScriptEditor::_set_breakpoint(REF p_script, int p_line, bool p_enabled) {
+	Ref<Script> script = Object::cast_to<Script>(*p_script);
+	if (script.is_valid() && (script->has_source_code() || script->get_path().is_resource_file())) {
+		if (edit(p_script, p_line, 0, false)) {
+			editor->push_item(p_script.ptr());
+
+			ScriptEditorBase *current = _get_current_editor();
+			if (ScriptTextEditor *script_text_editor = Object::cast_to<ScriptTextEditor>(current)) {
```

Comment:
No need to cast this to `ScriptTextEditor` any more.

Reply:
Fixed, thanks :+1: